java-generator/README.md

```diff
@@ -0,0 +1,35 @@
+# Java generator (early preview release)
+
+## Disclaimer
+
+This is an early preview of a Java generator from CRD and
+there might still be issues in the generated sources.
+Please give it a try and report any issue you might find.
+
+### Usage
+
+The `java-generator` is distributed as a tiny CLI application, you can obtain it by running:
+
+```
+mvn clean package -f cli/pom.xml
+```
+
+The produced executable will be `cli/target/java-gen` and it provides basic configuration options:
```

Comment:
Can `jib-maven-plugin` be added to make this a Docker image?

Reply:
I would say, not in this PR 🙂 
I think that we agreed on distributing this as simple Quarkus CLI, embedding it in a Docker image should be trivial after that.

Reply:
The scope of this PR (and of this project) is very far from creating a container image.

You could easily integrate tools such as JKube, Dekorate, Jib, buildpacks... in your project to generate the container image (if required) without polluting the client.